src/Evolution/Systems/CurvedScalarWave/PsiSquared.hpp

```diff
@@ -0,0 +1,35 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/CurvedScalarWave/TagsDeclarations.hpp"
+#include "Utilities/Gsl.hpp"
+
+/*!
+ * \brief The square of the scalar field \f$\Psi\f$.
```

Comment:
Looks like you accidentally put the dox comment on the namespace instead of the tag :)

Reply:
oups, my bad